SKILL.tsv

```diff
@@ -5614,12 +5614,12 @@ SKILL_20151102_005613	$Serpentine: Bleeding
 SKILL_20151102_005614	$Enemies hit by [Serpentine] have a 2% chance per attribute level to be afflicted with [Bleeding] for 8 seconds. Bleeding damage is proportional to the character's physical attack.
 SKILL_20151102_005615	$Gae Bulg: Enhance
 SKILL_20151102_005616	Increases the damage dealt on an enemy with [Gae Bulg] by 1% per attribute level.
-SKILL_20151102_005617
+SKILL_20151102_005617	
 SKILL_20151102_005618
 SKILL_20151102_005619	$Dragon Soar: Enhance
 SKILL_20151102_005620	$Increases the damage dealt on an enemy with [Dragon Soar] by 1% per attribute level.
 SKILL_20151102_005621	$Dragon Soar: Shock	
-SKILL_20151102_005622	
+SKILL_20151102_005622	$[Dragon Soar] gets a 5% chance per level to cause [Shock] for 10 seconds on enemies hit.
```

Comment:
"Afflicts [Shock] on enemies hit by [Dragon Soar] at a 5% chance per attribute level for 10 seconds."